x-pack/platform/plugins/shared/lens/public/react_embeddable/inline_editing/flyout.styles.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { UseEuiTheme } from '@elastic/eui';
+
+// styles needed to display extra drop targets that are outside of the config panel main area while also allowing to scroll vertically
+export const inlineFlyoutStyles = ({ euiTheme }: UseEuiTheme) => `
```

Comment:
Just a question, not something to change: I'm wondering why this was placed in its own file and not within the mount.tsx file (like in most changes I've seen).

Reply:
I think it was used twice within the code and then got removed. I'll restore it to the original file.